### ISSUE TITLE
storage: Accessibility issues

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#15192]
- [x] Progressbar widgets used as usage bars are missing accessible names. As the text to the site expresses the same thing, these widgets should probably use `aria-hidden="true"`

kkoukiou: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-valuetext_attribute is used for the progress bars to be accessible. This is done by PF itself.

- [ ] Journal entries lack correct role structure with rows being a part of grid/rowgroup/table/treegrid. (Related: #15191)
- [x] NFS mounts add button lacks labelling. This can be fixed by adding the word "Add" after the + icon. Alternatively, it should use aria-label (or aria-labelledby with an ID).

Addressed in other issues:
- iframe lacks title (#15188)
- html element lacks lang (#15189)

**Comment by marusak** [comment]
Only integrated journal issues are left. This will be fixed as part of #15191 as it uses the same renderer.
